pkg/network/infraconfigurators/masquerade.go

```diff
@@ -512,7 +511,10 @@ func getLoopbackAdrress(proto iptables.Protocol) string {
 	}
 }
 
-func portsUsedByLiveMigration() []string {
+func (b *MasqueradePodNetworkConfigurator) portsUsedByLiveMigration() []string {
+	if b.vmi.Status.MigrationTransport == v1.MigrationTransportUnix {
+		return []string{}
```

Comment:
nit: Any reason not to return just a `nil`?

Reply:
This has no particular meaning as there is nothing in this code that checks whether portsUsedByLiveMigration() is nil, so I don't mind